app/airq/commands/feedback.py

```diff
@@ -1,6 +1,7 @@
 import re
 import typing
 
+from flask_babel import gettext
```

Comment:
unused import? (and same for many other files)

Reply:
See above comment